cli/flox-rust-sdk/src/models/environment/generations.rs

```diff
@@ -653,10 +653,10 @@ impl AllGenerationsMetadata {
         (next_generation, history_ref)
     }
 
-    /// Switch the active marked generation to `next_generation`.
+    /// Switch the live marked generation to `next_generation`.
```

Comment:
(threading)

I feel more strongly about the first commit as I really don't want to overload activate

I feel less strongly that we should use "live" instead of "current." I can see some people having better intuition about what "current" means

Reply:
Fixed!